.gitpod.Dockerfile

```diff
@@ -66,6 +66,10 @@ RUN sudo apt-get update \
 # Source of original file https://github.com/gitpod-io/website/blob/main/src/routes/docs/languages/javascript.md
 
 # Use nvm to setup the version of node we want to use.
-RUN bash -c ". .nvm/nvm.sh     && nvm install 14     && nvm use 14     && nvm alias default 14"
+COPY .nvmrc ./
+RUN bash -c ". .nvm/nvm.sh \
+    && nvm install \
+    && nvm use"
+RUN echo "nvm use &>/dev/null" >> ~/.bashrc.d/51-nvm-fix
 
-RUN echo "nvm use default &>/dev/null" >> ~/.bashrc.d/51-nvm-fix
+RUN npm install -g npm@8
```

Comment:
Now the build works again, but I still get the shrinkwrap warning:

```bash
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
```

Running `node -v` and `npm -v` give these outputs:

```bash
gitpod /workspace/octoclairvoyant-webapp $ node -v
v14.18.1
gitpod /workspace/octoclairvoyant-webapp $ npm -v
6.14.15
```

So Gitpod still doesn't use the right version of npm when it starts up.

Reply:
Too soon, try now 😉

Reply:
That fixed it! 🥳